lab-goat/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = (dirPath, callback) => {
```

Comment:
Given that the entry point is `index.js` you shouldn't necessarily need to export it as modular functionality at this point. 

Also, what are you passing in as the dirPath and callback?